command/sidechain/withdraw/withdraw.go

```diff
@@ -37,9 +38,16 @@ func GetCommand() *cobra.Command {
 func setFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(
 		&params.accountDir,
-		sidechainHelper.AccountDirFlag,
+		polybftsecrets.DataPathFlag,
 		"",
-		"the directory path where validator key is stored",
+		polybftsecrets.DataPathFlagDesc,
+	)
+
+	cmd.Flags().StringVar(
+		&params.configPath,
+		polybftsecrets.ConfigFlag,
+		"",
+		polybftsecrets.ConfigFlagDesc,
 	)
```

Comment:
Make `DataPathFlag` and `ConfigFlag` mutually exclusive?

Reply:
done: https://github.com/0xPolygon/polygon-edge/pull/1251/commits/5f2f7453acce35cc3e7352c75442459c60908a31